griffon/commands/entities/osidb.py

```diff
@@ -201,6 +214,324 @@ def create_flaw(ctx, **params):
     return cprint(data, ctx=ctx)
 
 
+# flaw comments
+@flaws.group(help=f"{OSIDB_API_URL}/osidb/api/v1/flaws/<id>/comments", name="comments")
+@click.pass_context
+def flaw_comments(ctx):
+    """OSIDB Flaw Comments."""
+    pass
+
+
+@flaw_comments.command(name="get")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "comment_uuid", help="Comment UUID.", required=True)
+@query_params_options(
+    entity="Flaw Comment",
+    endpoint_module=osidb_api_v1_flaws_comments_retrieve,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(FlawComment))},
+    },
+)
+@click.pass_context
+@progress_bar
+def get_flaw_comment(ctx, flaw_id, comment_uuid, **params):
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+    data = session.flaws.comments.retrieve(flaw_id, comment_uuid, **params)
+    return cprint(data, ctx=ctx)
+
+
+@flaw_comments.command(name="list")
+@click.option(
+    "--flaw-id",
+    "flaw_id",
+    help="Flaw CVE-ID or UUID.",
+    required=True,
+)
+@query_params_options(
+    entity="Flaw Comments",
+    endpoint_module=osidb_api_v1_flaws_comments_list,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(FlawComment))},
+    },
+)
+@click.pass_context
+@progress_bar
+def list_flaw_comments(ctx, flaw_id, **params):
+    # TODO: handle pagination
+    # TODO: handle output
```

Comment:
all output should be handled in output.py

Reply:
Yes, I am aware of that, I am simply putting that there so it is clear which resources needs output handling implementation.